app/helpers/katalyst/navigation/frontend/builder.rb

```diff
@@ -40,6 +40,10 @@ def render_list(items)
           end
         end
 
+        def render_heading(heading)
+          tag.p heading.title
```

Comment:
if we're not using classes I wonder if there's some other semantic element we can use for headings
we could wrap the p in a `header` tag ?

Reply:
Updated to include the header tag